providers/base/bin/wifi_client_test_netplan.py

```diff
@@ -50,6 +50,18 @@ def print_cmd(cmd):
 NETPLAN_TEST_CFG = "/etc/netplan/99-CREATED-BY-CHECKBOX.yaml"
 
 
+def get_ubuntu_version():
```

Comment:
This should live in Checkbox support, not in an independent test script.

Python comes with some support for this in the stdlib (in the [`platform` module](https://docs.python.org/3/library/platform.html)), but it's inconsistent between Python versions unfortunately...